packages/components/time-select/src/time-select.vue

```diff
@@ -94,6 +116,10 @@ export default defineComponent({
     event: 'change',
   },
   props: {
+    use12h: {
+      type: Boolean,
```

Comment:
Maybe it's more appropriate to use this `use12Hours`

Reply:
hi @btea, you could insert a suggestion<ctrl + g> directly next time :)

Reply:
Okay, thanks for reminding, I didn’t know this usage before 😂

Reply:
I think we can judge by format, avoid add too many attributes

Reply:
> 

in my opinion, the time-select is just provide hours and minutes, i don't think we need more format. if users want to get more information or use more format they can use `time-picker`

Reply:
> 

I'm sorry, I thought it was time-picker